lib/archethic/p2p/client/connection.ex

```diff
@@ -118,21 +102,14 @@ defmodule Archethic.P2P.Client.Connection do
         data = %{
           ip: ip,
           port: port,
-          transport: transport,
-          node_public_key: node_public_key
+          transport: transport
         }
       ) do
     case transport.handle_connect(ip, port) do
       {:ok, socket} ->
         {:next_state, {:connected, socket}, data}
 
-      {:error, reason} ->
-        Logger.debug(
-          "Error during node connection to #{:inet.ntoa(ip)}:#{port} - #{reason} ",
-          node: Base.encode16(node_public_key)
-        )
-
-        MemTable.decrease_node_availability(node_public_key)
+      {:error, _} ->
```

Comment:
I removed this one because it means that if the connection is down we decrease the availability every 500 ms. But we need to do it just one time as we only change the availability bit from 1 to 0.
This update is done when we have an error on any request so I don't think we have to do it while trying to reconnect

Reply:
But there is not `decrease` in the module now.
If we want to do it once, we might put it when we enter in the state `disconnected`.

Reply:
Doing it there isn't enough ?
https://github.com/archethic-foundation/archethic-node/blob/4c9a983a4e081885c14583e3e27bc1371d195997/lib/archethic/p2p/client/default_impl.ex#L65

Reply:
I think it would be better to remove it from here and manage it in the connection (so it would be centralized)